### PR TITLE
feat: add configuration file of tier4 cicd

### PR DESCRIPTION
**Pull request by vios-fish**
## Description

Additional settings for testing Simulation in the CI/CD pipeline of TIER IV.

## Related links

https://github.com/autowarefoundation/autoware.ai/issues/2451

## Tests performed

The ability to run Autoware builds and tests using https://evaluation.tier4.jp/

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: .webauto-ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+version: 2
```
Comment (kenji-miyake) [inline .webauto-ci.yml]: [nits] Can you use `.yaml` instead of `.yml`? It's officially recommended.
http://yaml.org/faq.html

Comment (kenji-miyake) [inline .webauto-ci.yml]: Could you link some information about the format spec?

Comment (vios-fish) [reply .webauto-ci.yml]: Thanks!
fix it. [b2e7b8a](https://github.com/autowarefoundation/autoware/pull/397/commits/b2e7b8a42f100c361387ca12a65e70dbcc47d593)

Comment (vios-fish) [reply .webauto-ci.yml]: Sorry, the documentation is not available right now.
However, I am preparing to open a documentation site now, and I think I can explain it there.

**Review by kenji-miyake — APPROVED** [review]
Since it just adds a new file, I think there is no problem to merge this.
However, it's desired to share the format spec and evaluation result at a later date.

**Review by mitsudome-r — APPROVED** [review]
LGTM

**Pull request merged**